rust/protocol/src/identity_key.rs

```diff
@@ -15,31 +24,39 @@ use prost::Message;
 const ALTERNATE_IDENTITY_SIGNATURE_PREFIX_1: &[u8] = &[0xFF; 32];
 const ALTERNATE_IDENTITY_SIGNATURE_PREFIX_2: &[u8] = b"Signal_PNI_Signature";
 
+/// The public identity of a user, used in [IdentityKeyStore].
+///
+/// Wrapper for [PublicKey].
```

Comment:
The "used in IdentityKeyStore" seems a little too specific to me. Maybe just "A public key that represents the identity of a user"?

Reply:
Done!